Cargo.toml

```diff
@@ -41,6 +41,7 @@ num-bigint = { version = "0.4", optional = true }
 num-complex = { version = ">= 0.2, < 0.5", optional = true }
 rust_decimal = { version = "1.0.0", default-features = false, optional = true }
 serde = { version = "1.0", optional = true }
+smallvec = { version = "1.11.1", optional = true }
```

Comment:
Can this not use version 1.0?

Reply:
What do you mean? `"smallvec = 1.0"`

Reply:
Yes, in general we try to accept as broad ranges of optional dependency versions as possible so that we don't have to force users to use specific versions.

Reply:
Got it, made the change in 87cbd38229134e068df5c025a8843dbd7c51ebe1